### ISSUE TITLE
replace Sophia's carriage drive belt

### ISSUE THREAD

**Issue by DefProc** [DoESLiverpool/somebody-should#565]
The carriage drive belt (the y-axis belt, under the arm that moves the laser lens and final mirror) needs replacing, as it is failing, and is only being held together by the kevlar cords at two points(!). While it is currently still functional, it could fail in use at any time.

@DoESsean has emailed HPC to get some replacement belt sent through (and they still owe us for the belt for Gerald that we paid for and the couldn't supply), so *hopefully* we'll get a replacement through shortly.

![sophia-belt](https://user-images.githubusercontent.com/343000/30061365-fd587476-923e-11e7-86ef-720cb51a5725.jpg)

**Comment by DoESsean** [comment]
@jackie1050 reports that there are still a lot of slippages with Sophia, and it needs some serious cleaning.

**Comment by DefProc** [comment]
As far as I'm aware, we haven't had a reply from HPC yet (@DoESsean?). If that's the case, then we still need to chase them for a new belt, and we could also ask for some wheels for sophia (#198) which might help with ghosting.

**Comment by DoESsean** [comment]
I've never had a response from them to any email I've sent.

**Comment by DoESsean** [comment]
We need a laser carriage belt: LS3040

**Comment by jackie1050** [comment]
Darren has tried to order from 2 companies, but neither have been able to provide them.
The part numbers are MXL362 and MXL037
He has now put an enquiry in from a 3rd company
(I'll try to get the company names and record them here)

**Comment by amcewen** [comment]
Darren has got two shorter belts now (as that's all he's been able to source from the companies he's been talking to) so we'd need to cut and join them to use those.  As that's the case, I've just ordered a metre of 6.35mm [MXL belting](https://www.beltingonline.com/index.php?main_page=product_info&products_id=10002&cPath=40_740) from the people we used for the last belt for Sophia, as that'll only need one join.

**Comment by johncain39** [comment]
Yes, myself and Darran did this and the wheels about 3/4 weeks ago.

**Comment by amcewen** [comment]
The movement along that axis has been really stiff still, and @magman2112 noticed the belt is riding up and down the idler wheel (the one at the front-end of the cutter), suggesting the belt we bought might not be wide enough.  Reopening this so he can work out what size would work better so we can get another belt.

**Comment by magman2112** [comment]
It is actually that the belt is too wide for the idler wheel 6.3mm compared to 5.6mm. This causes the belt to ride up on the idler, jamming on one of the edges of the idler carrier bracket. I played around with the belt tension for a while but couldn’t get it to behave. 

I will have a look for another idler wheel or belt that matches either one or the other measurement.

**Comment by amcewen** [comment]
That makes more sense - I did wonder if the new belt would be too wide, but went with one of the standard sizes from beltingonline.com as it was hard to measure the correct size of the old belt.  They will do custom widths if we want to order something different...

**Comment by johncain39** [comment]
Hi Guys, when myself and Darren fitted the belt and wheels we measured the belt up against the old one and there was a slight difference. We didn’t know if the old
Belt had been burnt away or worn and decided to fit it any way as we also thought it was the right belt for the machine. The belt should not be moving up and down and I would presume this is down to the wheels on the cutting head being square and not chamfered and the cutting head moves quite a bit which I’m sure is not right?! If you do get a custom belt then I agree that the smaller width of 5.6mm would be better and should hopefully for correctly.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by magman2112** [comment]
I’ve been doing a bit of digging on the drive belt front and found a design on Thingiverse that may allow us to print our own idler wheel that fits the belt we have, though the design may need a bit of tweaking to fit.

https://www.thingiverse.com/thing:2395830

**Comment by N-Sinnott** [comment]
After Mike took measurements for the old idler wheel, we adjusted them for the new belt, before Julian and I modeled the [new design in OnShape](https://cad.onshape.com/documents/ec92bb69e144f4a1f9e3fdad/w/337ec295e7a834aa98ebf24f/e/57567ea30feab5c4e5705d11). We printed, then replaced the old wheel in Sophia with the 3D printed wheel. To my knowledge, Mike and Adrian adjusted other things in Sophia, and she is now fixed.
